### PR TITLE
Single object queries to use include and keys

### PR DESCRIPTION
**Pull request by jeremyjackson89**
Allow queries on specific objects to use the include and keys query parameters.

### CODE REVIEW

**Comment by drew-gross** [comment]
This code looks good, but would you mind added a test case for it? In `spec/ParseQuery.spec.js`

**Comment by jeremyjackson89** [comment]
Yeah, I will ASAP.

**Comment by jeremyjackson89** [comment]
Added the test cases.

File: src/Routers/ClassesRouter.js [hunk 1]
```diff
@@ -59,7 +59,24 @@ export class ClassesRouter extends PromiseRouter {
 
   // Returns a promise for a {response} object.
   handleGet(req) {
-    return rest.find(req.config, req.auth, req.params.className, {objectId: req.params.objectId})
+    let body = Object.assign(req.body, ClassesRouter.JSONFromQuery(req.query));
+    let options = {};
+    let allowConstraints = ['keys', 'include'];
```
Comment (flovilmart) [inline src/Routers/ClassesRouter.js]: can we move that up as a `const ALLOWED_GET_QUERY_KEYS`

Comment (jeremyjackson89) [inline src/Routers/ClassesRouter.js]: Done.

**Comment by kobyhastings** [comment]
@drew-gross @flovilmart when will this be released to the npm package? Thanks

**Pull request merged**